resdk/resources/collection.py

```diff
@@ -53,7 +53,7 @@ def __init__(self, slug=None, id=None,  # pylint: disable=redefined-builtin
     def data(self):
         """Lazy load ``data`` objects belonging to the collection."""
         if not self._data_hydrated:
-            self._data = self.resolwe.data.filter(id__in=','.join(map(str, self._data)))
+            self._data = self.resolwe.data.filter(id__in=','.join(map(str, self._data or [])))
```

Comment:
Wouldn't `[str(_) for _ in self._data]` instead of `map(str, self._data or []))` be more readable?